### PR TITLE
Monitoring CloudEvents example

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: insurance/modelmonitor.ipynb [hunk 1]
```diff
@@ -37,7 +39,48 @@
...
+    "    return inference, inference_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if USE_REMOTE_DEPLOYMENT:\n",
```
Comment (songole) [inline insurance/modelmonitor.ipynb:63]: The existing insurance example doesn't use cloudevents, right? It generates data. Is there a need to distinguish whether it is remote or local deployment?

File: insurance_cloudevents/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+   - **DKUBEUSERNAME** = {your dkube username}
+   - **MODELMONITOR_NAME** = {your model monitor name}
+   - **TOKEN** = {your dkube authentication token}
+   - **DKUBE_URL** = {your dkube url}
+   - **S3_KEY** = {MINIO access key}
+     - **S3_SECRET_KEY** = {MINIO access secret key}
+     - **S3_ENDPOINT** = {MINIO Endpoint}
+3. Run all the cells.
+
+## Section 2: Insurance Model Training
```
Comment (songole) [inline insurance_cloudevents/README.md]: Can we make this optional?

File: insurance_cloudevents/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+1. Add Code. Create Code Repo in Dkube with the following information
+  - Name: insurance
+  - Source: Git
+  - URL: https://github.com/oneconvergence/dkube-examples.git
+  - Branch : **monitoring**
+2. Create an IDE (JupyterLab)
+   - Use sklearn framework  
+3. Click Submit.
+4. Open Jupyterlab and from **workspace/insurance/insurance_cloudevents** open resources.ipynb and fill the following details in the first cell.
```
Comment (songole) [inline insurance_cloudevents/README.md]: We need to have instructions for both full dkube as well as minimal dkube. With monitor only deployment is somewhere else.

File: insurance_cloudevents/README.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+## Section 3: Modelmonitoring
+DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. This example cuurently support creating monitor using SDK. 
+
+- **Create Modelmonitor using SDK**
+1. From **workspace/insurance/insurance_cloudevents** open [modelmonitor.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance_cloudevents/modelmonitor.ipynb) and run all the cells. New model monitor will be created.
+2. Predict and Groundtruth dataset data will be generated by Data Generation step and will be utilised by modelmonitor.
+
+## Section 4: Data Generation
+1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance_cloudevents/data_generation.ipynb) notebook for making predictions with the deployemnt endpoint and generate groundtruth datasets.
+2. In 1st cell, Update Frequency according to what you set in Modelmonitor for Drift. For eg: for 5 minutes, specify it as `5m` and to specify the frequency in hours use `5h` for 5 hours interval.
```
Comment (songole) [inline insurance_cloudevents/README.md:82]: Where is the labelled data stored? We should not require AWS to run this.

Unfortunately, we don't have append option to our datasets.  local dataset option in the insurance example only works with full dkube.

Comment (songole) [reply insurance_cloudevents/README.md:82]: We can keep local dataset option only for full dkube for now

File: insurance_cloudevents/README.md (hunk 2 above)
Comment (riteshkarval) [reply insurance_cloudevents/README.md]: Without training, the deployment will not be possible.

File: insurance_cloudevents/README.md (hunk 3 above)
Comment (riteshkarval) [reply insurance_cloudevents/README.md]: okay, we can add the instructions for this case.

File: insurance_cloudevents/README.md (hunk 4 above)
Comment (riteshkarval) [reply insurance_cloudevents/README.md:82]: The predict data i.e cloudevents are being stored in {deployemnt-id}/YYYY/MM/DD/HH and the labelled data will be get stored in {deployemnt-id}/label_data in the same modelmonitor bucker.

File: insurance/modelmonitor.ipynb (hunk 1 above)
Comment (riteshkarval) [reply insurance/modelmonitor.ipynb:63]: This is not the cloudevent example. It's our  existing example, which can support imported also.

**Pull request merged**